### PR TITLE
buildnumberadjust 1.0.0

### PR DESCRIPTION
**Pull request by stefandevo**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by viktorbenei** [comment]
Thanks @stefandevo for the PR! As this is a new step we'll do a more thorough check.

File: steps/buildnumberadjust/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  and you already have a certain build number to start with)
+website: https://github.com/stefandevo/bitrise-buildnumberadjust-step
+source_code_url: https://github.com/stefandevo/bitrise-buildnumberadjust-step
+support_url: https://github.com/stefandevo/bitrise-buildnumberadjust-step/issues
+published_at: 2017-04-30T19:59:35.758143112+02:00
+source:
+  git: https://github.com/stefandevo/bitrise-buildnumberadjust-step.git
+  commit: 4188b654c2ff941f2297cbecc9bf35cb148fd76b
+type_tags:
+- script
```
Comment (viktorbenei) [inline steps/buildnumberadjust/1.0.0/step.yml]: @godrei I believe this should be `utility`, right?

Comment (stefandevo) [inline steps/buildnumberadjust/1.0.0/step.yml]: Sure... Is there a list with type tags to use in which cases?

Comment (viktorbenei) [inline steps/buildnumberadjust/1.0.0/step.yml]: @stefandevo I have an internal one, waiting for @godrei for the public one. Not sure if that was already published - we're in the middle of normalizing the tags, for better step discovery (main categories will be used on the UI too, soon ;))

Comment (stefandevo) [inline steps/buildnumberadjust/1.0.0/step.yml]: Ok, clear. Meantime, changed it.

Comment (viktorbenei) [inline steps/buildnumberadjust/1.0.0/step.yml]: Thank you! :)

**Comment by godrei** [comment]
Hi @stefandevo ,
thanks for the step PR!

I agree with your new project_type_tag, for future reference, you can read more about step grouping [here](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-grouping-convention).

Could you please use hyphen (`-`) separated words for your step id, [reference](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-id-naming-convention). I would recommend to use `build-number-adjust`.

To apply your step id changes, please re-share your step, by:

- close this PR
- delete the related branch in your steplib fork
- update your fork
- share your step again

**Comment by viktorbenei** [comment]
Sorry @stefandevo for the change requests, we're in the middle of restructuring the related docs & guidelines, to provide the best experience for both developers and users of the steps.

If we can help with anything just let us know!

**Pull request closed without merging**